Doc/c-api/init.rst

```diff
@@ -2291,6 +2291,18 @@ The C-API provides a basic mutual exclusion lock.
 
    .. versionadded:: 3.13
 
+.. c:function:: int PyMutex_IsLocked(PyMutex *m)
+
+   Returns one if the mutex *m* is currently locked, zero otherwise.
```

Comment:
For booleans like this one, usually we just say "non-zero" rather than one:

```suggestion
   Returns non-zero if the mutex *m* is currently locked, zero otherwise.
```